### PR TITLE
Add Sebastian's information to AboutUs

### PR DESCRIPTION
**Pull request by sebastiantoh**
To resolve part of #6

### CODE REVIEW

File: docs/team/sebastiantoh.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+layout: page
+title: Sebastian Toh's Project Portfolio Page
+---
+
+## Project: StonksBook (WIP)
+
+AddressBook - Level 3 is a desktop address book application used for teaching Software Engineering principles. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
```
Comment (Asthenosphere) [inline docs/team/sebastiantoh.md]: Should change to StonksBook here as well.

Comment (sebastiantoh) [reply docs/team/sebastiantoh.md]: Thanks for pointing that out!

**Review by Asthenosphere — APPROVED** [review]
Nice work!

**Review by hakujitsu — APPROVED** [review]
LGTM!

**Pull request merged**